_posts/2025-07-25-A-real-use-case-with-OVOS-and-Hivemind.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "A real use case with OVOS and Hivemind"
+excerpt: "A practical example of OVOS and Hivemind running locally on standard hardware, designed for home automation and basic AI assistant tasks — especially suited for supporting people with disabilities."
+coverImage: "/assets/blog/common/cover.png"
+date: "2025-07-25T00:00:00.000Z"
+author:
+  name: "Menne Bos"
+  picture: "https://github.com/MenneBos.png"
+ogImage:
+  url: "https://github.com/MenneBos/ovos-skill-HomeyFlowTrigger/blob/main/Media/Sat_kitchen_small.png"
+---
+
+# Why Home Automation with OVOS matters
+
+Over the past five years, I’ve visited many people with disabilities in their homes and witnessed firsthand how much effort it can take to control everyday things like lights, heating, doors, curtains, and appliances. Often, this leads to less comfort than what’s technically possible. Giving people access to voice-activated home automation isn't just convenient—it offers them greater independence, comfort, and peace of mind.
+
+Such a system typically consists of two core components: a home automation gateway and a voice assistant. OVOS (Open Voice OS) is a voice-first assistant that integrates seamlessly with home automation systems. Its full focus on voice means that every part of the system—from wakeword to intent—is designed with spoken interaction in mind.
+
+With OVOS, I can fully customize and control intents for things like switching on lights, playing music, or interacting with an AI assistant. That level of control is often missing in commercial systems. Sometimes, when you ask them to close the curtains, they might suggest where to buy new ones instead.
+
+## OVOS and Hivemind filling Real Needs
+
+I'm using an OVOS server with Hivemind satellites, which allows me to adapt the system to virtually any use case. These lightweight, low-cost satellites make it feasible to have a voice assistant in every room. What’s more, I can easily customize the enclosures to suit specific situations—like fitting them into a bathroom, or attaching one to a wheelchair. Thanks to their low power consumption, the satellites can even be powered directly from a wheelchair battery.
```

Comment:
@JarbasAl We should consider adding links where appropriate, so anyone getting to the blog post from a search engine can immediately go to the relevant site to set up their own

Reply:
Also, I love the solution of integrating a voice satellite directly with a wheelchair @MenneBos !